assets/js/components/HostDetails/HostSummary.stories.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
I don't think we need a story for this @EMaksy 
We need a story in the `HostDetails` view storybook, with a +3 IP addresses scenario

Reply:
Removed the story and added a new scenario :)